pkgs/applications/misc/googleearth-pro/default.nix

```diff
@@ -80,7 +80,7 @@ mkDerivation rec {
     mkdir $out
     mv usr/* $out/
     rmdir usr
-    mv * $out/
+    mv etc opt $out/
```

Comment:
Alternatively, we could `tar --exctract -C some_dir` and `mv some_dir/* $out` to ensure that everything gets copied (but not the temporary build files from Nix)? Hardcoding `etc opt` is a bitrot risk.

Reply:
Good idea, done.